conda/unix.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
     - autodoc
     - bokeh
     - breathe
+    - cairosvg
```

Comment:
Excuse me if I'm missing something, but how exactly are we using `cairosvg`?

As far as I can know, `cairosvg` is for converting SVG images into png/pdf form (`drawSVG` has APIs that use cairo) but converting to these formats prevents any animations from rendering. Non-animated SVGs should be able to be displayed just fine alongside normal jupyter widgets with the `DrawingWidget` object, which skips the extra step of converting to another filetype.

Reply:
Yes, you are right, we are not using cairosvg. But drawsvg was generating a multi-line warning in red, which did not look good. Now that we are moving things to the python file I can possibly remove it.
Thanks for pointing it out.